persistent/Database/Persist/Types/Base.hs

```diff
@@ -157,14 +157,14 @@ data EntityDef = EntityDef
     -- ^ Optional comments on the entity.
     --
     -- @since 2.10.0
-    , entitySpan :: !(Maybe Span)
+    , entitySourceSpan :: !(Maybe SourceSpan)
```

Comment:
this is internal and should not be used by folks, but keeping the old name would be *less* breaking. up to you on these record field names